### DISCUSSION TITLE
Some native iOS/iPadOS apps I worked A LOT to make nice were HORRIBLY WRONGLY AND CORRUPTLY removed from the Apple app stores after I promoted liberation of Palestine, so I am interested in assisting to FORCE iOS apps to run on Android phones.

### DISCUSSION THREAD

**Discussion by 01GOD** [darlinghq/darling#1528] [Q&A]
Hi! 

Some native iOS/iPadOS apps I worked A LOT to make nice were HORRIBLY WRONGLY AND CORRUPTLY removed from the Apple app stores after I promoted liberation of Palestine, so I am interested in assisting to FORCE iOS apps to run on Android phones. 

Yes, sadly Apple has been assisting israel to commit a GENOCIDE in Palestine!!! From the "app store connect" conversations, it seemed as though the israeli mafia (that masquerades as a government) had intentionally planted some "israel affiliate" agents in the "app store connect" review department to intentionally censor the Apple app stores for israel. 

That and how some corrupt scumbags at Apple HORRIBLY CORRUPTLY AND DECEITFULLY attempted to cheat the EU's new anti-monopoly regulations are reason enough to FORCE iOS apps to run on Android now!!! I recently thankfully made a Swift app WITH a GUI and WITHOUT any BS interference or "signing" BS from the MCP known as the tim cook Apple. That's right, a GUI Swift app without Xcode and without ANY signing and without ANY interference from Apple or tracking by Apple. That was done with terminal and a text editor. So now that needs to happen on iOS/iPadOS!!! 

Consider this post the hammer from that historical Apple Macintosh commercial SMASHING through tim cook's greedy, deceitful face on that screen. 

Hopefully the entire team (saurik and so on) will join in on this to finally FULLY make that happen!!! I propose an ANGEL standing on that horned platypus and stabbing it with a spear of LIGHT as the project SYMBOL because I totally HATE any of those vile horned mascots of bsd related things, and because it is a VERY appropriate SYMBOL for slaying the corrupt beast that Apple's app store monopolies have sadly become.

**Comment by 01GOD** [comment]
PS: By the way, although I have done some embedded, I am still learning ARM64 Assembly. I have used Hopper before and done some disassembly with a terminal app, but I would appreciate coaching on an IRC channel or similar. Sadly discord (as MLP fans will recognize as very aptly named) was founded by corrupt, deceitful "israel affiliates" also, and certainly NOT with a nice agenda. It's sadly a data farm that is probably used by "israel affiliates" to train ML models, and sadly "israel affiliates" also censor discord (which they had a problem doing on some IRC networks).

**Comment by mrolappe** [comment]
This is not the place for you to release strong negative emotions. Thank you for showing more respect in the future.

**Comment by LubosD** [comment]
I, the founder of this project, also consider myself affiliated with Israel :smirk: With this I close this discussion.
